pyproject.toml

```
@@ -25,8 +25,8 @@ classifiers = [
   "Topic :: Utilities",
 ]
 dependencies = [
-    "dissect.cstruct>=3.0.dev,<4.0.dev",
-    "dissect.util>=3.0.dev,<4.0.dev",
+    "dissect.cstruct>3,<5",
+    "dissect.util>=3.0,<4.0",
```

```suggestion
    "dissect.util>2,<4",
```
`>3` to get optional `3.0.dev` versions, `<4` to prevent getting `4.0.dev` versions ;)